src/contact.rs

```diff
@@ -1030,7 +1030,7 @@ fn split_address_book(book: &str) -> Vec<(&str, &str)> {
         .chunks(2)
         .into_iter()
         .filter_map(|mut chunk| {
-            let name = chunk.next().unwrap();
+            let name = chunk.next().unwrap_or_default();
```

Comment:
i'm not 100% sure what's going on here, it uses itertools afaik.  @dignifiedquire wrote this so may have a better idea.

Reply:
i think the first unwrap() is always safe as otherwise the callback would not be called. the second unwrap() is only done after checking. i will change that.